app/src/test/java/org/simple/clinic/summary/linkId/LinkIdWithPatientViewLogicTest.kt

```diff
@@ -58,14 +58,14 @@ class LinkIdWithPatientViewLogicTest {
   }
 
   @Test
-  fun `when the view is created, the identifier must be displayed`() {
+  fun `when the view is created, patient name must be displayed`() {
     // when
-    whenever(patientRepository.patientImmediate (patientUuid)).thenReturn(patient)
+    whenever(patientRepository.patientImmediate(patientUuid)).thenReturn(patient)
     setupController()
     uiEvents.onNext(LinkIdWithPatientViewShown(patientUuid, identifier))
 
     // then
-    verify(ui, atLeastOnce()).renderIdentifierText(identifier)
+    verify(ui).renderPatientName(patientName)
     verifyNoMoreInteractions(ui, uiActions)
   }
```

Comment:
We can delete this test since it being covered now in the new effect handler and UI renderer tests.